framework/test/jest/integration/specs/modules/chain/blocks/util.spec.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+describe('integration test (blocks) - process', () => {
+	describe('loadBlocksWithOffset() - no errors', () => {
+		it.todo('should load block 2 from db: block without transactions');
+
+		it.todo('should load block 3 from db: block with transactions');
+	});
+
+	describe('loadBlocksOffset() - block/transaction errors', () => {
```

Comment:
Maybe split these into `describe('loadBlocksOffset() - block errors')` and `describe('loadBlocksOffset() - transaction errors')`